src/views/Home.js

```diff
@@ -50,27 +50,47 @@ export class Home extends Component {
         );
     }
 
+    _reloadComponent() {
+        this.props.setAuthToken(undefined);
+        this._renderButtons()
+    }
+
     _renderButtons () {
         if (!this.props.token) {
             return (
                 <View>
+                    <Text style={styles.titleText}>Welcome to Eventador</Text>
+                    <Text style={styles.baseText}>Let's find you something to do tonight</Text>
+                    <Text style={styles.baseText}>{'\n\n\n'}</Text>
```

Comment:
😆

Reply:
If you're wondering, the standard way to do this is to add `paddingBottom` to the style.

Reply:
ty ty 🙏

Reply:
I _knew_ I was doing a bad